docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -88,7 +88,24 @@ and any other application-specific secrets.
 
 ### Protect against brute force attacks
 
-Introduce a measure against brute force login attacks (captcha, etc.).
+Consider introducing a measure against brute force login attacks, like CAPTCHA. Adjust timeout limits to your needs:
+
+When using the "forgot password" feature, a token is created which expires if the user doesn't click the password reset
+link that gets mailed to them in time. The time before it expires is set in the parameter
+`ibexa.site_access.config.default.security.token_interval_spec`. By nature this feature must be available to users
+before they have logged in, including would-be attackers. If an attacker uses this feature with someone else's email
+address, the attacker does not receive the email. But they could still try to guess the password reset link. That's why
+this interval should be as short as possible. 5 minutes is often enough.
+
+Ibexa DXP allows you to create and send invitations to create an account in the frontend as a customer, the Back Office
+as an employee, or the Corporate Portal as an organisation member. You can send invitations to individual users or in
```

Comment:
```suggestion
as an employee, or the Corporate Portal as a company member. You can send invitations to individual users or in
```
OR
```suggestion
as an employee, or the Corporate Portal as a bussines partner. You can send invitations to individual users or in
```

To be consistent with the rest of documentation on this topic and TOC in Back Office